api_client/python/timesketch_api_client/aggregation.py

```diff
@@ -214,6 +219,18 @@ def lazyload_data(self, refresh_cache=False):
         # TODO: Implement a method to refresh cache.
         return self.resource_data
 
+    @property
+    def parameters(self):
```

Comment:
Why not just make the attribute public? You are just returning it here, so no need for the property

Reply:
The main reason is that if it's public you can change it easily... but the change won't be reflected in the saved object.... by having it as a property it is now a read-only attribute, by then implementing a setter later I can then make sure that if you modify the parameters they will be saved in the actual aggregation object, that is the object will be updated on the server side.

Reply:
ack, SG